src/Domain.LinnApps/Boms/ChangeRequest.cs

```diff
@@ -216,5 +216,27 @@ public void MakeLive(Employee appliedBy, IEnumerable<int> selectedBomChangeIds,
                 }
             }
         }
+
+        public bool CanPhaseIn() => this.ChangeState == "ACCEPT";
+
+        public void PhaseIn(LinnWeek week, IEnumerable<int> selectedBomChangeIds)
+        {
+            if (this.CanPhaseIn() && selectedBomChangeIds != null)
+            {
+                if (this.BomChanges != null)
```

Comment:
could this if just be an additioanl && clause in the above one? Just to reduce nesting